### ISSUE TITLE
Blacklist nog correctly working

### ISSUE THREAD

**Issue by CubeStick** [Elkosscom/poe_gwennen_gambler#4]
Loving the program so far.
However I made a list of all the replica uniques and put them in the blacklist text file.
But it's still showing them in the program (not all tho, it's weird).
I tried stopping and starting the program, closing it and re-opening it.
Nothing seems to work, it's not updating it.
I even used the program itself to add it to the blacklist.
It does get added in the blacklist file but it's not applying it for some reason.
![bug](https://user-images.githubusercontent.com/56218378/127747434-4d37010d-ccee-455b-a732-9a83a4a64c3b.png)

**Comment by Elkosscom** [comment]
Can you check the config if UseBlacklist is set to True? If it is set to False, it will always show False on the Blacklisted field.
```
[Prices]
UnlinkedOnly = True
MinimumMeanChaosValue = 0.0
IgnoreFatedUniques = True
UseBlacklist = True
MinItemLevelRestriction = True
SortBy = Expected Chaos
```

**Comment by Elkosscom** [comment]
Fixed in 1.2, uploading it now. Issue was with uppercase letters.
